include/net/nrf_cloud_cell_pos.h

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#ifndef NRF_CLOUD_CELL_POS_H_
+#define NRF_CLOUD_CELL_POS_H_
+
+/** @file nrf_cloud_cell_pos.h
+ * @brief Module to provide nRF Cloud cellular positioning support to nRF9160 SiP.
+ */
+
+#include <zephyr.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/** @defgroup nrf_cloud_cell_pos nRF Cloud cellular positioning
+ * @{
+ */
+
+/** @brief Cell-based location request type */
+enum cell_based_location_type {
+	CELL_LOC_TYPE_INVALID = -1,
+	CELL_LOC_TYPE_SINGLE,
+	CELL_LOC_TYPE_MULTI /* Not yet supported */
+};
+
+/**@brief Request a cell-based location query from nRF Cloud.
+ *
+ * @param type        Type of cell-based location to request.
+ * @param request_loc If true, cloud will send location to the device.
+ *                    If false, cloud will not send location to the device.
+ * @return 0 if successful, otherwise a (negative) error code.
+ */
+int nrf_cloud_request_cell_location(enum cell_based_location_type type, const bool request_loc);
+
+/**@brief Gets most recent location from cellular positioning request.
+ *
+ * @param lat Pointer where last cellular positioning latitude is to be copied.
+ * @param lon Pointer where last cellular positioning longitude is to be copied.
+ * @return 0 if successful, otherwise a (negative) error code.
+ */
+int nrf_cloud_get_last_cell_location(double *const lat, double *const lon);
+
+/**@brief Processes cellular positioning data received from nRF Cloud.
+ *
+ * @param buf Pointer to data received from nRF Cloud.
+ * @param buf_len Buffer size of data to be processed.
+ *
+ * @return 0 if successful, otherwise a (negative) error code.
```

Comment:
add return value 1.
a return value of 1 indicates that no cell pos data was found.  this is not considered an error.